Code/Editor/EditorFramework/Assets/Implementation/AssetBrowserModel.cpp

```diff
@@ -59,6 +60,14 @@ struct FileComparer
       sSortB = b.m_sAbsFilePath.GetAbsolutePath().GetFileNameAndExtension();
     }
 
+    if (m_bSortByRecentlyUsed && pInfoA && pInfoB)
```

Comment:
This will only work if all items are assets. What you probably want to do is:
1. If both assets, do this.
2. If one asset AND the asset has a valid LastAccess time, its smaller in order.